dynamique/schema-dynamique.json

```
@@ -66,8 +66,8 @@
         },
         {
             "name": "etat_pdc",
-            "description": "TODO",
-            "example": "TODO",
+            "description": "etat_pdc caractérise l’état de fonctionnement du point de recharge : est-il en service ou hors service ? En l’absence d’information, etat_pdc sera égal à ‘inconnu’.",
```

```suggestion
            "description": "`etat_pdc` caractérise l’état de fonctionnement du point de recharge : est-il en service ou hors service ? En l’absence d’information, etat_pdc sera égal à ‘inconnu’.",
```

C'est bien de mettre les noms de champs entre  \`\`, ça fait qu'ils apparaissent dans la doc comme `ceci`.